docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -7,38 +7,40 @@
-in Voltage mode with the PWM protocol or the DSHOT protocols from the flight controller.
+This tutorial is meant to walk you through the process of performing basic setup and testing for controlling a Vertiq module
+with a flight controller using :ref:`Standard PWM <hobby_standard_pwm>` or :ref:`DSHOT <hobby_dshot>`. This tutorial covers only the basics of testing 
+that the flight controller can make the module spin with the appropriate protocol, not the setup of any additional flight controller peripherals like an RC controller
+or a GPS. This tutorial should take you from a totally fresh module and flight controller, to a module that can be controlled
+with the :ref:`Standard PWM <hobby_standard_pwm>` or :ref:`DSHOT <hobby_dshot>` protocols from the flight controller.
 
-This tutorial covers setup and testing on **Ardupilot using Mission Planner** and **PX4 using QGroundControl**. It covers using
-the **PWM** or **DSHOT** protocol, since the setup process is largely similar for both protocols. In the places where there are differences,
-the proper steps for both types of protocol have been recorded.
+This tutorial covers setup and testing on `ArduCopter <https://ardupilot.org/copter/>`_ using `Mission Planner <https://ardupilot.org/planner/>`_ and 
+`PX4 <https://px4.io/>`_ using `QGroundControl <http://qgroundcontrol.com/>`_. It covers using both the :ref:`Standard PWM <hobby_standard_pwm>` additonal :ref:`DSHOT <hobby_dshot>` 
+protocols, since the setup process is largely similar for both protocols. In the places where there are differences, the proper steps for both types of protocol are explained.
 
-These instructions are applicable to both the **Vertiq 8108** and the **Vertiq 2306** with speed firmware. Though they have
-different form factors and some different configuration parameters available, all of the configurations and setup for
-PWM or DSHOT are the same between them. The images of hardware and from the Control Center used are from an 8108, but the
-procedure is applicable to the 2306 as well.
+These instructions are applicable to any Vertiq module using speed firmware. Though they have different form factors and some different configuration parameters available, 
+the configuration and setup for basic control with Standard PWM or DSHOT is the same for all Vertiq speed modules. This example uses a Vertiq 8108 for demonstration purposes, 
+but it is not specific to the 8108.
 
 Hardware Setup
 ==============
-This example uses a Vertiq 8108 as the motor and a Pix32 from Holybro as the flight controller. Since this tutorial is
-focused only on the motor, the only additional perpheral we are using along with the flight controller is a safety switch.
+This example uses a Vertiq 8108 and a Pix32 from Holybro as the flight controller. Since this tutorial is
+focused specifically on setting up the Vertiq module, the only additional perpheral that is used along with the flight controller is a safety switch.
```

Comment:
I changed instances where we refer to our module as a "motor" to module, since that is more correct and matches the language we use on our website. Our modules are a motor plus an integrated ESC.